packages/payload/src/admin/components/views/API/index.tsx

```diff
@@ -173,7 +173,7 @@ function createURL(url: string) {
 
 export const API: React.FC<EditViewProps> = (props) => {
   const { apiURL } = props
-  const { i18n } = useTranslation()
+  const { i18n, t } = useTranslation(['authentication', 'general', 'version'])
```

Comment:
Passing the array is optional, all scopes are preloaded by default. Just a note.

Reply:
Updated!